packages/frontend/src/components/card/match.tsx

```diff
@@ -29,9 +29,7 @@ export function MatchCard({
       className={`${className} rounded-lg border w-full border-gray-300 p-2 flex items-center bg-p-grey-200`}
     >
       <div className="mr-4">
-        <div className="image-container">
-          <Profile imgSrc={profilePic} />
-        </div>
+        <div className="image-container">{<Profile imgSrc={profilePic} />}</div>
```

Comment:
?????????

Reply:
Idk, how that happened. Interesting though.